spec/ttml1.xml

```diff
@@ -1132,6 +1161,9 @@ which all foreign namespace elements and attributes have been removed. Therefore
 exceptional reporting of this false negative does not impact the formal assessment
 of <emph>Document Instance</emph> validity.</p>
 </note>
+<note role="clarification">
+<p>The schemas referenced by this specification are designed for use after the pruning steps (1)-(3) specified by <specref ref="doctypes"/> have been applied.</p>
```

Comment:
I think I'd prefer "intended" rather than "designed" here but that's a minor point.